notebooks/6.Incorporating_Related_Time_Series_dataset_to_your_Predictor.ipynb

```diff
@@ -55,10 +55,14 @@
    "source": [
     "%load_ext autoreload\n",
     "%autoreload 2\n",
-    "from util.fcst_utils import *\n",
+    "\n",
     "import warnings\n",
     "import boto3\n",
-    "import s3fs\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import util\n",
```

Comment:
We are importing both. Is there a way to import only the functions we need.

Reply:
@akhilrazdan 
I need more clarification in this feedback. In the notebook, all these modules are used.

Reply:
Addressed by the latest commit.